src/bootstrap-table.js

```diff
@@ -846,6 +856,8 @@
                 }
                 var aa = getItemField(a, name),
                     bb = getItemField(b, name),
+                    aa_position = getItemField(a, 'position'),
+                    bb_position = getItemField(b, 'position'),
```

Comment:
I don't know why here use position field, If haven't use this field? Please provide a working jsfiddle with your changed code, thanks.